BDArmory/Weapons/Missiles/MissileBase.cs

```diff
@@ -1156,6 +1169,33 @@ protected void UpdateAntiRadiationTarget()
             if (targetGPSCoords != Vector3d.zero)
                 TargetPosition = VectorUtils.GetWorldSurfacePostion(targetGPSCoords, vessel.mainBody);
         }
+        private bool setInertialTarget = false;
+
+        public Vector3d UpdateInertialTarget()
+        {
+            Vector3 TargetCoords_;
+            if (!setInertialTarget)
+            {
+                driftSeed = new Vector3(UnityEngine.Random.Range(-1, 1) * inertialDrift, UnityEngine.Random.Range(-1, 1) * inertialDrift, UnityEngine.Random.Range(-1, 1) * inertialDrift);
```

Comment:
`inertialDrift * Random.insideUnitSphere` (https://docs.unity3d.com/ScriptReference/Random-insideUnitSphere.html) might be more appropriate for this.